.github/workflows/build-package-docs.yaml

```
@@ -92,7 +92,7 @@ jobs:
       working-directory: build-directory/docs/docsite
 
     - name: Create latest symlink
-      if: github.event.inputs.latest-symlink
+      if: ${{ github.event.inputs.latest-symlink == 'true'}}
```

No need for the mustache. Also, could you check if a simple [`fromJSON()`](https://docs.github.com/en/actions/learn-github-actions/expressions#fromjson) works here?
```suggestion
      if: fromJSON(github.event.inputs.latest-symlink)
```

---

thanks @webknjaz I've tested and it works